### ISSUE TITLE
GBIF checklist dataset page order

### ISSUE THREAD

**Issue by rukayaj** [gbif/portal-feedback#4456]
Hi, is it possible to reorder the entries in e.g. https://www.gbif.org/species/search?dataset_key=38f91564-30ff-47b0-aa6b-cf3b59f0fca7&origin=SOURCE&advanced=1  so they are alphabetical?

**Comment by MortenHofft** [comment]
No, the API (which the website use) do not support that https://www.gbif.org/developer/species#searching
